deployment/debian-package-x64/pkg-src/conf/jellyfin

```diff
@@ -19,10 +16,16 @@
 JELLYFIN_DATA_DIRECTORY="/var/lib/jellyfin"
 JELLYFIN_CONFIG_DIRECTORY="/etc/jellyfin"
 JELLYFIN_LOG_DIRECTORY="/var/log/jellyfin"
+JELLYFIN_CACHE_DIRECTORY="/var/cache/jellyfin"
+
 # Restart script for in-app server control
-JELLYFIN_RESTART_SCRIPT="/usr/lib/jellyfin/restart.sh"
-# Additional options for the binary
-JELLYFIN_ADD_OPTS=""
+JELLYFIN_RESTART_OPT="--restartpath /usr/lib/jellyfin/restart.sh"
+
+# [OPTIONAL] ffmpeg binary paths
+#JELLYFIN_FFMPEG_OPTS="-ffmpeg /usr/bin/ffmpeg -ffprobe /usr/bin/ffprobe"
```

Comment:
```suggestion
#JELLYFIN_FFMPEG_OPTS="--ffmpeg /usr/bin/ffmpeg --ffprobe /usr/bin/ffprobe"
```
I presume single-dash options are working as a backwards compatibility thing, so I don't think we should use them in new (or edited) scripts.

Reply:
No they don't any more, good catch!

Reply:
Done.